typings/index.d.ts

```diff
@@ -80,7 +81,7 @@ declare module 'discord.js' {
   //#region Classes
 
   export class Activity {
-    constructor(presence: Presence, data?: object);
+    constructor(presence: Presence, data?: unknown);
```

Comment:
I mean at this point we might as well go deeper and use the typings here, no? We don't need to `unknown` them

Reply:
We need to merge this regardless because of the Snowflake mismatch we already have when someone is using master.

But yes, this opens the possibility of filling this out further (not really to anyone's benefit though since... you don't really instantiate those yourself).